xcm/pallet-xcm-benchmarks/src/fungible/benchmarking.rs

```diff
@@ -107,6 +107,25 @@ benchmarks_instance_pallet! {
 		// TODO: Check sender queue is not empty. #4426
 	}
 
+	reserve_asset_deposited {
+		let (trusted_reserve, transferable_reserve_asset) = T::TrustedReserve::get()
+			.ok_or(BenchmarkError::Skip)?;
+
+		let assets: MultiAssets = vec![ transferable_reserve_asset ].into();
```

Comment:
just to refresh my memory on this as it was a while ago since I last looked at it: XCM benchmarks are all per-asset right? and then later on, elsewhere we sum all the weights per asset?

Reply:
Unsure what you mean by that, but this `transferable_reserve_asset` is provided by configuration, and so if the chain maintainer wants to benchmark different assets, they'll have to specify it via the `TrustedReserve` associated type.